composer.json

```diff
@@ -7,22 +7,24 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=7.2,<8.0-dev",
+        "ext-dom": "*",
         "ext-openssl": "*",
         "robrichards/xmlseclibs": "^3.0.4",
         "simplesamlphp/saml2": "3.2.*",
         "symfony/dependency-injection": "^4.4",
         "symfony/framework-bundle": "^4.4",
         "symfony/templating": "^4.4",
-        "ext-dom": "*"
+        "twig/twig": "^2"
```

Comment:
In order to run the tests, we need Twig to be able to render the metadata template